merkle/merkle_queue.go

```diff
@@ -0,0 +1,325 @@
+package merkle
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/kyokan/plasma/util"
+	"math"
+	"sync"
+)
+
+const (
+	maxDepth = 17
```

Comment:
shouldn't this be 16?

Reply:
If we want to support max 65536 (2^16) leaves than the depth should be 17.

Reply:
👍